Sources/MapboxNavigation/StatusView.swift

```diff
@@ -46,6 +46,39 @@ public class StatusView: UIControl {
         }
     }
     
+    var statuses: [Status] = []
+
+    /**
+    `Status` is a struct which stores information to be displayed by the `StatusView`
+     */
+    public struct Status {
+        public var identifier: String
+        let title: String
+        var spinner: Bool = false
+        let duration: TimeInterval
+        var animated: Bool = true
+        var interactive: Bool = false
+        var priority: Priority
```

Comment:
These properties also need to be public (and documented) in order for developers to set them.